packages/core/src/plugins/logStore/network/LogStoreNetworkPlugin.ts

```diff
@@ -41,12 +44,22 @@ export class LogStoreNetworkPlugin extends LogStorePlugin {
 	private readonly propagationResolver: PropagationResolver;
 	private readonly propagationDispatcher: PropagationDispatcher;
 	private readonly reportPoller: ReportPoller;
+	private readonly notaryServer: ProxiedWebServerProcess;
+	private readonly proxyRequestProver: NetworkProver;
 
 	private metricsTimer?: NodeJS.Timer;
 
 	constructor(options: PluginOptions) {
 		super(options);
 
+		this.notaryServer = new ProxiedWebServerProcess(
+			'notary',
+			WEBSERVER_PATHS.notary(),
```

Comment:
Right now, the Notary and Prover servers that instantiate automatically are the Python WebServers.
They should be the Rust Servers .... that actually contain the core logic.

Obviously.